admin-dev/themes/new-theme/js/pages/catalog/product/specific-price-form-handler.ts

```diff
@@ -215,7 +215,10 @@ class SpecificPriceFormHandler {
    * @private
    */
   private reinitializeDatePickers() {
-    $('.datepicker input').datetimepicker({format: 'YYYY-MM-DD'});
+    $('.datepicker input').datetimepicker({
+      format: 'YYYY-MM-DD HH:mm:ss',
+      sideBySide: true,
```

Comment:
Why the side-by-side option? What happens when it remains false as default, the time picker is under the calendar? Isn't it better as it uses less space on the screen?

Reply:
It doesn't work at all if it's not there, `sideBySide: true,` needs to be there for the datetime format to work. See https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/js/app/utils/datepicker.js